modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -173,6 +174,14 @@ public void processOpts() {
         } else {
             setIgnoreImportMapping(defaultIgnoreImportMappingOption());
         }
+
+        if(additionalProperties.containsKey(CodegenConstants.NOT_NULL_JACKSON_ANNOTATION)){
+            this.setNotNullJacksonAnnotation((Boolean.parseBoolean( additionalProperties.get(CodegenConstants.NOT_NULL_JACKSON_ANNOTATION).toString())));
+        }
+    }
+
+    protected void setNotNullJacksonAnnotation(boolean notNullJacksonAnnotation) {
+        this.notNullJacksonAnnotation = notNullJacksonAnnotation;
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator